lib/signs/utilities/messages.ex

```diff
@@ -23,41 +26,84 @@ defmodule Signs.Utilities.Messages do
         sign_config,
         current_time,
         alert_status,
-        scheduled,
+        first_scheduled_departures,
+        last_scheduled_departures,
+        most_recent_departure,
         service_status
       ) do
+    config =
```

Comment:
This is good pulled out this way, so we only have to do the transformation once.